File: CS/T142225/Controllers/HomeController.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.Mvc;
using MVCxGridViewDataBinding.Models;

namespace T142225.Controllers
{
    public class HomeController : Controller
    {
        //
        // GET: /Home/

      
        public ActionResult Index()
        {
            return View();
        }
        public ActionResult Index2()
        {
            return View();
        }
        public ActionResult TabPage1()
        {
            var model = DateTime.Now;
            return PartialView("PartialView1", model);
        }
        [ValidateInput(false)]
        public ActionResult TabPage2()
        {
            var model = MyModel.GetModelList();
            return PartialView("PartialView2", model);
        }
        public ActionResult TabPage3()
        {
            var model = new MyModel(3);
            return PartialView("PartialView3", model);

        }
        public ActionResult PageControlPartial()
        {
            return PartialView();
        }
        [ValidateInput(false)]
        public ActionResult GridViewPartial()
        {
            var model = MyModel.GetModelList();
            return PartialView("PartialView2", model);
        }
    }
}

File: VB/T144225VB/Controllers/HomeController.vb
Imports T144225VB.MVCxGridViewDataBinding.Models

Namespace T144225VB
    Public Class HomeController
        Inherits System.Web.Mvc.Controller

        Public Function PageControlPartial() As ActionResult
            Return PartialView()
        End Function
        Public Function Index() As ActionResult
            Return View()
        End Function
        Public Function Index2() As ActionResult
            Return View()
        End Function
        Public Function TabPage1() As ActionResult
            Dim model = DateTime.Now
            Return PartialView("PartialView1", model)
        End Function
        <ValidateInput(False)> _
        Public Function TabPage2() As ActionResult
            Dim model = MyModel.GetModelList()
            Return PartialView("PartialView2", model)
        End Function
        Public Function TabPage3() As ActionResult
            Dim model = New MyModel(3)
            Return PartialView("PartialView3", model)
        End Function
        <ValidateInput(False)> _
        Public Function GridViewPartial() As ActionResult
            Dim model = MyModel.GetModelList()
            Return PartialView("PartialView2", model)
        End Function
    End Class
End Namespace
